app/models/concerns/startdate_beyond_osra_establishment_validation.rb

```diff
@@ -0,0 +1,12 @@
+module StartdateBeyondOsraEstablishmentValidation
+	include ValidatorHelpers
+
+	OSRA_ESTABLISHMENT_DATE = Date.new(2012,04,01)
```

Comment:
I'm going to disagree: `OSRA_ESTABLISHMENT_DATE` is a part of the business logic of our application, used by multiple entities. Something like this does belong in a central shared location. On the other hand, pagination params are specific not only to individual models, but are also relevant only to HTML rendering by Rails. If our app gets set up to e.g. also return JSON structures to web services or mobile clients, these constants lose all meaning. Country selection falls into the same category - it's something unique to Rails sponsor views. I'd prefer not to keep such constants mixed in with those that belong to the core functionality of OSRA.